lib/Command/RenderCommand.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\RST\Command;
+
+use Doctrine\RST\Builder;
+use Doctrine\RST\Configuration;
+use Doctrine\RST\Formats\Format;
+use http\Exception\RuntimeException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\FormatterHelper;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+use function assert;
+use function getcwd;
+use function str_starts_with;
+
+class RenderCommand extends Command
+{
+    public function __construct()
+    {
+        parent::__construct('render');
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setHelp('This command allows you to render your documentation...')
+            ->addOption(
+                'deleteCache',
+                'd',
+                InputOption::VALUE_NONE,
+                'Delete the cache before rendering?',
+            )
+            ->addOption(
+                'format',
+                'f',
+                InputOption::VALUE_OPTIONAL,
```

Comment:
Why would the value be optional? `--format` alone does not have much meaning, right?

```suggestion
                InputOption::VALUE_REQUIRED,
```